src/Standards/BestIt/Sniffs/Commenting/AbstractDocSniff.php

```diff
@@ -186,9 +186,9 @@ private function isSimpleText(array $possCommentToken): bool
      *
      * @param int $startPosition
      *
-     * @return int
+     * @return int|bool
```

Comment:
Now you see, for what the decorator was used ... no type security now :(

Reply:
Yes not nice ARGH. But a decorator must have the same definitions and this is not possible in this case.

Reply:
Do you remember the discussion in the Devtreff this year where I said that the Union Type should never be used but there is still a situation where it is helpful?

Just saying ...

https://wiki.php.net/rfc/union_types_v2